core/src/test/java/org/mskcc/cbio/portal/scripts/TestCutInvalidCases.java

```diff
@@ -44,14 +42,12 @@
  * JUnit test for CutInvalidCases class.
  */
 public class TestCutInvalidCases {
-
-	// TODO Fix this test following the change in data file
-	@Ignore
+    
 	@Test
     public void testCutInvalidCases() throws Exception {
 		// TBD: change this to use getResourceAsStream()
-        File casesExcludedFile = new File("target/test-classes/cases_excluded_test.txt");
-        File dataFile = new File("target/test-classes/cna_test.txt");
+        File casesExcludedFile = new File("src/test/resources/cases_excluded_test.txt");
```

Comment:
I don't remember what was the initial reason to refer the files in target/test-classes/ instead of src/test/resources, but I think this is the right way to refer to test files.